src/containers/Admin/EditTab/EditTab.tsx

```diff
@@ -609,6 +629,110 @@ const EditTab = (): JSX.Element => {
                 </Heading2>
             </div>
             <PosterMobilityAlert />
+            <div>
+                <div className="edit-tab__secondary-title">
+                    Mobilitetstilbud
+                </div>
+                <ResponsiveReactGridLayout
+                    key={breakpoint}
+                    cols={COLS}
+                    layouts={LAYOUT}
```

Comment:
Burde man ha delt `LAYOUT` i to? En for bruk i denne ResponsiveReactGridLayout og en for den andre? Tenker det ikke er så farlig om vi tar sikte på å fjerne ResponsiveReactGridLayout her ganske snart.